tests/foundry/foundry.toml

```diff
@@ -2,3 +2,5 @@
 src = 'src'
 out = 'out'
 extra_output = ['storageLayout', 'abi', 'evm.methodIdentifiers', 'evm.deployedBytecode.object']
+rpc_endpoints = { optimism = "https://optimism.alchemyapi.io/v2/...", mainnet = "${RPC_MAINNET}" }
+
```

Comment:
Do we need these rpc endpoints set for the tests, or just some of them? If it's just some of them, maybe we can somehow set it only for those tests?

Reply:
We can delete the `mainnet` entry but we have to update the test. Do you want me to do that?